admin/spam/serializers.py

```diff
@@ -0,0 +1,45 @@
+from website.project.model import Comment, User
+from modularodm import Q
+
+
+def serialize_comment(comment, full=False):
+    comment = {
+        'id': comment._id,
+        'author': User.load(comment.user._id),
+        'date_created': comment.date_created,
+        'date_modified': comment.date_modified,
+        'content': comment.content,
+        'has_children': bool(getattr(comment, 'commented', [])),
+        'modified': comment.modified,
+        'is_deleted': comment.is_deleted,
+        'reports': serialize_reports(comment.reports),
+        'node': comment.node,
+    }
+    comment.update(
+        category=comment['reports'][0]['category'],
+    )
+    return comment
+
+
+def retrieve_comment(id, full_user=False):
+    return serialize_comment(Comment.load(id), full=full_user)
+
+
+def serialize_comments():
```

Comment:
This is inconsistent with the other serialization functions because it queries for the objects it is serializing (rather than taking the objects as input). Better to make this consistent with the other functions.